plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java

```diff
@@ -238,6 +245,56 @@ public static void main(String[] args)
         }
     }
 
+    public static final class IcebergAzureQueryRunnerMain
+    {
+        private IcebergAzureQueryRunnerMain() {}
+
+        public static void main(String[] args)
+                throws Exception
+        {
+            String azureContainer = requireNonNull(System.getenv("AZURE_CONTAINER"), "Environment variable AZURE_CONTAINER must be provided");
+            String azureAccount = requireNonNull(System.getenv("AZURE_ACCOUNT"), "Environment variable AZURE_ACCOUNT must be provided");
+            String azureAccessKey = requireNonNull(System.getenv("AZURE_ACCESS_KEY"), "Environment variable AZURE_ACCESS_KEY must be provided");
```

Comment:
Did you avoid `ABFS_CONTAINER`, `ABFS_ACCOUNT` and `ABFS_ACCESS_KEY` names in ci.yml intentionally? 
Also, using the same system properties (`hive.hadoop2.azure-abfs-*`) in ci.yml instead of environment variables might be better so that we can reuse them in IntelliJ easily.

Reply:
> might be better so that we can reuse them in IntelliJ easily

Excellent idea